benefits/core/session.py

```diff
@@ -143,34 +142,24 @@ def language(request):
 
 
 def logged_in(request):
-    """Check if the current session has an OAuth token."""
-    return bool(oauth_authorized(request))
+    """Get the user's status of having logged in with OAuth from the request's session, or None"""
+    return bool(request.session.get(_LOGGED_IN))
 
 
 def logout(request):
     """Reset the session claims and tokens."""
-    update(request, oauth_claims=[], oauth_authorized=False, enrollment_token=False)
-
-
-def oauth_authorized(request):
-    """Get the oauth authorization status from the request's session, or None"""
-    return request.session.get(_OAUTH_AUTHORIZED)
-
-
-def oauth_claims(request):
-    """Get the oauth claims from the request's session, or None"""
-    return request.session.get(_OAUTH_CLAIMS)
+    OAuthSession(request, claims_result=ClaimsResult())
+    update(request, logged_in=False, enrollment_token=False)
 
 
 def oauth_extra_claims(request):
     """Get the extra oauth claims from the request's session, or None"""
-    claims = oauth_claims(request)
-    if claims:
-        f = flow(request)
-        if f and f.uses_claims_verification:
-            claims.remove(f.claims_request.eligibility_claim)
-            return claims
-        raise Exception("Oauth claims but no flow")
+    oauth_session = OAuthSession(request)
+    eligibility_claim = oauth_session.claims_request.eligibility_claim
+    requested_extra_claims = [claim for claim in oauth_session.claims_request.claims_list if claim != eligibility_claim]
```

Comment:
I wonder if it's worth adding methods to `ClaimsVerificationRequest` to give us the `extra_claims` list instead of us having to calculate it

Reply:
Agreed this makes a lot of sense to move to the `cdt_identity` session wrapper

Reply:
Created https://github.com/Office-of-Digital-Services/django-cdt-identity/issues/50